src/api/app/models/update_notification_events.rb

```diff
@@ -20,15 +20,18 @@ def create_events
           event.save!
         rescue ActiveRecord::StatementInvalid => e
           retries -= 1
-          retry if retries.positive?
+          if retries.positive?
+            Airbrake.notify("Failed to create Event : #{type.inspect}: #{data} #{e}")
+            retry
+          end
           Airbrake.notify("Failed to create Event : #{type.inspect}: #{data} #{e}")
         rescue StandardError => e
-          if Rails.env.test?
-            # make debug output useful in test suite, not just showing backtrace to Airbrake
-            Rails.logger.error "ERROR: #{e.inspect}: #{e.backtrace}"
-            Rails.logger.info e.inspect
-            Rails.logger.info e.backtrace
-          end
+          # if Rails.env.test?
```

Comment:
I think you should remove those or leave them be...

Reply:
I removed those lines.